python/ray/serve/controller.py

```diff
@@ -139,11 +140,26 @@ def _put_serve_snapshot(self) -> None:
             entry["version"] = backend_info.version or "Unversioned"
             # TODO(architkulkarni): When we add the feature to allow
             # deployments with no HTTP route, update the below line.
-            # Or refactor the route_prefix logic in the Deployment class now.
+            # Or refactor the route_prefix logic in the Deployment class.
             entry["http_route"] = route_prefix or f"/{deployment_name}"
             entry["status"] = "RUNNING"
-            entry["start_time"] = 0
+            entry["start_time"] = backend_info.start_time_ms or 0
```

Comment:
uh this is bad for frontend parsing... this will probably show up as 1970/01/01 00:00 in the UI cc @kombuchafox is this field nullable?